tap.js

```diff
@@ -37,8 +59,9 @@
             this.el.addEventListener('touchend', this, false);
             this.el.addEventListener('touchcancel', this, false);
 
-        } else if (e.type === 'mousedown') {
+        } else if (e.type === 'mousedown' && this.whichbutton(e) === 1) {
```

Comment:
It looks like Safari doesn't support event.buttons in the MDN doc, have you tested that this falls back gracefully?